scripts/templates/detector_res.tf.j2

```diff
@@ -0,0 +1,53 @@
+{% set id = name | replace(' ', '_') -%}
+resource "signalfx_detector" "{{ id }}" {
+  name = format("%s %s", local.detector_name_prefix, "{{ module | capitalize }} {{ name | capitalize }}")
+
+  program_text = <<-EOF
+    {%- for key, signal in signals.items() %}
+      {%- if 'metric' in signal %}
+    {{ key }} = data('{{ signal.metric }}', filter=${module.filter-tags.filter_custom})
+        {%- if aggregation | default(true) %}${var.{{ id }}_aggregation_function}{%- endif %}
+        {%- if transformation | default(true)%}${var.{{ id }}_transformation_function}{%- endif %}
+      {%- endif %}
+      {%- if 'formula' in signal %}
+    {{ key }} = {{ signal.formula }}.publish('{{ key }}')
+      {%- endif %}
+    {%- endfor %}
+    {%- for key, rule in rules.items() %}
+      {%- set severity = key | lower %}
+      {%- if '>' in rule.comparator %}
+        {%- set reverse_comparator = '<' %}
+      {%- else %}
+        {%- set reverse_comparator = '>' %}
+      {%- endif %}
+      {%- if '=' in rule.comparator %}
+        {%- set negative_comparator = reverse_comparator %}
+      {%- else %}
+        {%- set negative_comparator = reverse_comparator ~ '=' %}
+      {%- endif %}
+    detect(when(signal > ${var.{{ id }}_threshold_{{ severity }}})).publish('{{ severity | replace('critical', 'crit') | replace ('warning', 'warn') | upper }}')
```

Comment:
```suggestion
    detect(when(signal > ${var.{{ id }}_threshold_{{ severity }}})
```

Reply:
we need this to keep the existing convention CRIT and WARN

Reply:
nah, this comment is paired with the next one :) (I moved the code in the else condition)

Reply:
I think I already resolved in recently pushed commit please tell me if not.